--- a/data/noise.desktop
+++ b/data/noise.desktop
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-Name=Noise
+Name=Music
 GenericName=Music Player
 Comment=Listen to music
 Exec=noise %U
@@ -18,4 +18,4 @@
 
 [Desktop Action AboutDialog]
 Exec=noise --about
-Name=About Noise
+Name=About Music
--- a/src/Noise.vala
+++ b/src/Noise.vala
@@ -56,7 +56,7 @@
         build_version = Build.VERSION;
         build_version_info = Build.VERSION_INFO;
 
-        program_name = PROGRAM_NAME;
+        program_name = _("Music");
         exec_name = "noise";
 
         app_copyright = "2012-2014";
